libs/cpprestsdk/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2018 Bruno Randolf (br1@einfach.org)
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cpprestsdk
+PKG_VERSION:=2.10.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-git.tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL:=https://github.com/Microsoft/cpprestsdk
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=4ebbc832e6abaf3a3abd9f70bb9f9556fcd6b6645239c76e84d9201f8f805678
+
+PKG_MAINTAINER:=Bruno Randolf <br1@einfach.org>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=license.txt
+
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DEPENDS:=websocketpp
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_INSTALL:=1
+CMAKE_SOURCE_DIR:=Release
+CMAKE_OPTIONS += -DWERROR:Bool=OFF
```

Comment:
why are you turning this off? is upstream broken?

Reply:
sorry this was a leftover. i removed it.